app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -336,6 +338,11 @@ public void requestPermissions(int actionId) {
                         throw new IllegalArgumentException("Unknown Action");
                 }
             }
+
+            @Override
+            public void permissionDeniedToast(int actionId) {
+                Toast.makeText(getContext(), getAskAgainSnackBarString(actionId ), Toast.LENGTH_LONG).show();
```

Comment:
I think this is not what the spec describes?

Reply:
fixed